js/index.js

```diff
@@ -1,7 +1,48 @@
 // Iteration 1: Names and Input
+const hacker1 = "Victoria";
+console.log(`The driver's name is ${hacker1}`);
 
+const hacker2 = "Sami";
+console.log (`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length){
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+  }
+  else if (hacker1.length < hacker2.length){
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+  }
+  else {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!.`);
+  }
 
-// Iteration 3: Loops
\ No newline at end of file
+// Iteration 3: Loops
+
+let driverName = "";
+for (let i=0; i<hacker1.length; i++){
+ driverName += " " +hacker1[i].toUpperCase(); 
+}
+
+console.log(driverName.slice(1));
+
+let navigatorName = "";
+for (let i=hacker2.length-1; i >= 0; i--){
+  navigatorName += hacker2[i];
+}
+console.log(navigatorName);
+
+let arrayOfNames =[hacker1, hacker2];
+arrayOfNames.sort();
```

Comment:
This can give  a bug especially when it comes to capitalization of letters because a != A. The best method here would be a .localeCompare() method where a == A to prevent any bugs arising from capitalization of letters in a name.